Resources/Prototypes/GameRules/roundstart.yml

```
@@ -421,10 +421,10 @@
   parent: BaseGameRule
   components:
   - type: BasicStationEventScheduler
-    minimumTimeUntilFirstEvent: 1200 # 20 mins
+    minimumTimeUntilFirstEvent: 1200 # 45 mins # starcup: 20 > 45 mins for parity with standard scheduler
```

```suggestion
    minimumTimeUntilFirstEvent: 2700 # 45 mins # starcup: 20 > 45 mins for parity with standard scheduler
```
I should've pointed this out explicitly in my first comment, apologies. 1200 is 20 minutes.